### PR TITLE
Document the requirement to refresh if adding items

### PR DESCRIPTION
**Pull request by andydotxyz**
### Checklist:

- [x] Tests included.
- [x] Lint and formatter run with no errors.
- [x] Tests all pass.

### CODE REVIEW

File: container/theme.go [hunk 1]
```diff
@@ -24,6 +24,9 @@ type ThemeOverride struct {
 // Containers will be traversed and all child widgets will reflect the theme in this container.
 // This should be used sparingly to avoid a jarring user experience.
 //
+// If the content `obj` of this theme override is a container and items are later added to the container
```
Comment (dweymouth) [inline container/theme.go:27]: maybe rephrase - "are later added to the container **or any sub-containers**"?

**Review by dweymouth — APPROVED** [review]
looks good with one optional suggestion

File: container/theme.go (hunk 1 above)
Comment (andydotxyz) [reply container/theme.go:27]: Thanks, I pushed that in as well
